pkgs/top-level/all-packages.nix

```
@@ -7269,13 +7269,15 @@ with pkgs;
 
   gocryptfs = callPackage ../tools/filesystems/gocryptfs { };
 
-  godot = callPackage ../development/tools/godot {};
+  godot_4 = callPackage ../development/tools/godot_4 { };
 
-  godot-export-templates = callPackage ../development/tools/godot/export-templates.nix { };
+  godot = callPackage ../development/tools/godot_3 { };
```

```suggestion
  godot = callPackage ../development/tools/godot/3 { };
```
How about we move this into a subdirectory or use the following:

```suggestion
  godot = callPackage ../development/tools/godot/3.nix { };
```

---

I think the subdirectory way is better, as the godot 4 variants will differ from godot 3. Also, thanks for the review.

---

Consider making the directories `godot/godot_3` and `godot/godot_4`.